### PR TITLE
feat: implement periodic docker container cleanup

### LINKED ISSUE #120: implement periodic docker container removal
Sometimes, for unknown reasons, some docker containers are not removed.

To fix this we currently use a cronjob to periodically remove them.
Ideally, we want to implement that in our codebase.

### PR DESCRIPTION
**Pull request by Nfsaavedra**
Closes #120

### CODE REVIEW

File: gitbugactions/test_executor.py [hunk 1]
```diff
@@ -14,14 +22,54 @@ def __init__(
...
+            ):
+                return
+
+            docker = DockerClient.getInstance()
+
+            def cleanup():
+                for container in docker.containers.list(
+                    all=True, filters={"ancestor": runner_image, "status": "exited"}
+                ):
+                    container.remove()
```
Comment (andre15silva) [inline gitbugactions/test_executor.py:50]:
```suggestion
                    container.remove(force=True)
```

Maybe forcing here is even better?

**Review by andre15silva — APPROVED** [review]

**Comment by andre15silva** [comment]
LGTM, see comment before merging

File: gitbugactions/test_executor.py (hunk 1 above)
Comment (Nfsaavedra) [reply gitbugactions/test_executor.py:50]: Yap that sounds like a good idea

**Comment by Nfsaavedra** [comment]
I'll rerun the integration tests just to be 100% that the force does not break anything, and then I'll merge

**Pull request merged**